core/domain/email_manager.py

```diff
@@ -263,7 +308,20 @@ def _send_bulk_mail_in_transaction(instance_id=None):
 
 
 def send_mail_to_admin(email_subject, email_body):
-    """Sends email to admin."""
+    """Send email to admin.
+
+    The email is sent to the ADMIN_EMAIL_ADDRESS set in config.
+
+    Args:
+        email_subject: str. Subject of the email.
+        email_body: str. Body (message) of the email.
+
+
+    Raises:
+        Exception: Exceptions from _send_email, send_mail such as
```

Comment:
Interesting, I see what you mean, and am torn about this. On the one hand, this function is a public interface function and so it's important to document the exceptions properly. On the other hand, this also suggests that callers of this function should do the same thing, which just ends up becoming unmanageable.

Let's err on the side of non-duplication. I suggest removing this section altogether, since this function itself does not raise any exceptions. I know this goes against what I suggested originally (sorry). I'll bring this up with the tech leads as well and ask what the style preference is, and we can make changes in a follow-up PR if needed.